.github/workflows/ci.yml

```diff
@@ -69,7 +69,6 @@ jobs:
       run: |
           python -m pip install --upgrade pip
           pip install requests
-          python download-wgpu-native.py
           pip uninstall -q -y requests
```

Comment:
I think you're literally installing and then deinstalling requests here

Reply:
Aah, I saw that, wondered wtf that was, and forgot to get back to it. In the diff it's clear what the intention was 😅  Will fix